scripts/rmc-regression.sh

```diff
@@ -24,6 +24,10 @@ check-cbmc-viewer-version.py --major 2 --minor 5
 ./x.py build -i --stage 1 library/std ${EXTRA_X_PY_BUILD_ARGS}
 ./scripts/setup/build_rmc_lib.sh
 
+# Build tool for linking RMC pointer restrictions
+rustup toolchain install nightly
+cargo +nightly build --release --manifest-path src/tools/rmc-link-restrictions/Cargo.toml 
```

Comment:
why do we need nightly?

Reply:
The `smallvec` dependency of cbmc requires it, trying without that now

Reply:
Maybe create a symbolic link to `src/rmc-compiler/rust-toolchain.toml` inside `src/tools/rmc-link-restrictions/`. We want to lock in a specific version.

I would use a link so we track the same version.

The only caveat is that I think you need to execute cargo from the folder where the toolchain file is.